src/bp/ui-studio/src/web/views/FlowBuilder/diagram/nodes/SaySomethingContents/index.tsx

```diff
@@ -41,11 +41,11 @@ const SayNodeContent: FC<Props> = props => {
     const { image, title, subtitle } = item
 
     return (
-      <div key={index} className={style.contentImgWrapper}>
-        {image && <div style={{ backgroundImage: `url('${image}')` }} className={style.img}></div>}
-        <div className={style.textWrapper}>
-          {title && <span className={style.primaryText}>{title}</span>}
-          {subtitle && <span className={style.secondaryText}>{_.truncate(subtitle, { length: 25 })}</span>}
+      <div key={index} className={localStyle.contentImgWrapper}>
```

Comment:
I know you probably didnt write this. Can you change the key to something else.than the index for better react performance